themes/ra-theme/layouts/partials/ra-logo.svg

```diff
@@ -14,6 +14,7 @@
       stroke: none;
       fill: black;
       text-transform: uppercase;
+      text-decoration: none;
```

Comment:
You should NOT have diffs from other PR's in your PR.  Please REBASE you commits so merges do not appear here.